modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -68,59 +68,74 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 +
 // Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
 
-. Convert the `journal.conf` file to base64:
+. Convert the `journal.conf` file to base64 by running the following command:
 +
 [source,terminal]
 ----
-$ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
+$ export jrnl_cnf=$( cat journald.conf | base64 -w0 )
 ----
 
-. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step.
+. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step. Following is the sample command for worker:
 +
-For example:
+[NOTE]
+====
+You can create a MachineConfig object for both master and worker machine types.
+====
 +
 [source,terminal]
 ----
-$ cat > /tmp/40-worker-custom-journald.yaml <<EOF
+$ cat << EOF > ./99-worker-journald-configuration.yaml <1>
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 50-corp-journald
+    machineconfiguration.openshift.io/role: worker <2>
+  name: 99-worker-journald-configuration <3>
 spec:
   config:
     ignition:
-      version: 3.1.0
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
```

Comment:
Please check with QE.  I believe the Ignition config spec version for 4.7 is 3.2.0.

<https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks>

Reply:
This is already tested and approved by QE for both master and worker as you can see above. I can recheck if that's needed

Reply:
Yes,   "version": "3.2.0" in 00-worker.  It is better to use 3.2.0